src/include/common/object_pool.h

```diff
@@ -25,7 +26,7 @@ struct ByteAllocator {
    * Reuse a reused chunk of memory to be handed out again
    * @param reused memory location, possibly filled with junk bytes
    */
-  void Reuse(T *reused) { PELOTON_MEMSET(reused, 0, sizeof(T)); }
+  void Reuse(T *reused) {}
```

Comment:
More of a style thing, do we need this function?

Reply:
Some one could perceivably want to do something with the reused pointer.

We don't have a use case for now.